src/frontend/csrs-portal/proxy.conf.json

```diff
@@ -0,0 +1,8 @@
+{
+    "/api/*": {
+      "target": "http://jag-csrs-portal-public-f0b5b6-dev.apps.silver.devops.gov.bc.ca/",//"http://localhost:8081/",
```

Comment:
confused, why are you proxying to the remote dev portal?

Reply:
Remote dev portal was replaced by local.